src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -347,7 +347,7 @@ file that was distributed with this source code.
 
 {% block form_row %}
     {% set show_label = show_label|default(true) %}
-    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}" id="sonata-ba-field-container-{{ id }}">
+    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}{% if attr.row_class|default('') %}{{ attr.row_class|default('') }}{% endif %}" id="sonata-ba-field-container-{{ id }}">
```

Comment:
looks like a space is missing, imo this could result in `form group has errorcustom-class`, but should be:
`form group has error custom-class`
```suggestion
    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}{% if attr.row_class|default('') %} {{ attr.row_class|default('') }}{% endif %}" id="sonata-ba-field-container-{{ id }}">
```